awstest/modules/apigateway/module.ftl

```diff
@@ -297,4 +297,138 @@
+                "Namespace" : "mockacct-shared",
+                "Settings" : {
+                    "Registries": {
+                        "openapi": {
+                            "EndPoint": "account-registry-abc123",
+                            "Prefix": "openapi/"
+                        }
+                    }
+                }
+            },
+            {
+                "Type" : "Builds",
+                "Scope" : "Products",
+                "Namespace" : "mockedup-integration-aws-apigateway-cfdistro",
+                "Settings" : {
+                    "COMMIT" : "123456789#MockCommit#",
+                    "FORMATS" : ["openapi"]
+                }
+            },
+            {
+                "Type" : "Settings",
+                "Scope" : "Products",
+                "Namespace" : "mockedup-integration-aws-apigateway-cfdistro",
+                "Settings" : {
+                    "API_ACCESSKEY": "1234567890#MockAPIKey",
+                    "apigw": {
+                        "Internal": true,
+                        "Value": {
+                            "Type": "lambda",
+                            "Proxy": false,
+                            "BinaryTypes": ["*/*"],
+                            "ContentHandling": "CONVERT_TO_TEXT",
+                            "Variable": "LAMBDA_API_LAMBDA"
+                        }
+                    }
+                }
+            }
+        ]
+        blueprint={
+            "Tiers" : {
+                "app" : {
+                    "Components" : {
+                        "apigatewaycfdistro" : {
+                            "apigateway" : {
+                                "Instances" : {
+                                    "default" : {
+                                        "DeploymentUnits" : ["aws-apigateway-cfdistro"]
+                                    }
+                                },
+                                "Certificate": {},
+                                "CloudFront": {},
+                                "Image" : {
+                                    "Source" : "none"
+                                },
+                                "IPAddressGroups" : [ "_global" ],
+                                "Profiles" : {
+                                    "Testing" : [ "apigatewaycfdistro" ]
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "TestProfiles" : {
+                "apigatewaycfdistro" : {
+                    "apigateway" : {
+                        "TestCases" : [ "apigatewaycfdistro" ]
+                    }
+                }
+            },
+            "TestCases" : {
+                "apigatewaycfdistro" : {
+                    "OutputSuffix" : "template.json",
+                    "Structural" : {
+                        "CFN" : {
+                            "Resource" : {
```

Comment:
Should probably have a test to see if the Cloudfront distribution is included as a resource

Reply:
Done